webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/Repository.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FormGroup, TextInput, Popover } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import { translate as __ } from '../../../../../common/I18n';
+
+const Repository = ({
+  repo,
+  handleRepo,
+  repoGpg,
+  handleRepoGpg,
+  isLoading,
+}) => (
+  <>
+    <FormGroup
+      label={__('Repository')}
+      fieldId="reg_smart_proxy"
+      labelIcon={
+        <Popover
+          bodyContent={__(
+            "Repository URL / details, for example for Debian OS family: 'deb http://deb.example.com/ buster 1.0', for Red Hat OS family: 'http://rpm.example.com/'"
+          )}
+        >
+          <button
+            className="pf-c-form__group-label-help"
+            onClick={e => e.preventDefault()}
+          >
+            <HelpIcon noVerticalAlign />
+          </button>
+        </Popover>
+      }
+    >
+      <TextInput
+        id="reg_repo"
+        value={repo}
+        type="text"
+        onChange={handleRepo}
+        isDisabled={isLoading}
+      />
+    </FormGroup>
+    <FormGroup
+      label={__('Repository GPG key')}
+      fieldId="reg_smart_proxy"
+      labelIcon={
+        <Popover bodyContent={__('GPG key for the repository')}>
```

Comment:
"If packages are GPG signed, the public key can be specified here to verify the packages signatures. It needs to be specified in the ascii form with the -----BEGIN PGP PUBLIC KEY BLOCK----- header."

The last sentence is my assumption, also perhaps we can just call it `with the pgp public key header`.